README.md

```diff
@@ -69,6 +69,11 @@ Please make sure your contributions adhere to our coding guidelines:
  * Commit messages should be prefixed with the package(s) they modify.
    * E.g. "eth, rpc: make trace configs optional"
 
+### Submitting an issue
+
+If you come across a bug, you can run `geth bug` to open a GitHub issue draft
```

Comment:
It may be worth clarifying that if they were running the node with `docker run` they should run this command the same way